SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/LayeredLBStrategy/LayeredLBStrategyTest.cpp

```diff
@@ -0,0 +1,412 @@
+#include "LoadBalancing/LayeredLBStrategy.h"
+#include "Tests/AutomationCommon.h"
+#include "Tests/AutomationEditorCommon.h"
+#include "Tests/TestDefinitions.h"
+#include "Engine/Engine.h"
+#include "GameFramework/GameStateBase.h"
+#include "EngineClasses/SpatialWorldSettings.h"
+#include "LoadBalancing/GridBasedLBStrategy.h"
+#include <vector>
+#include <utility>
+#include "SpatialGDKTests/SpatialGDK/LoadBalancing/GridBasedLBStrategy/TestGridBasedLBStrategy.h"
+#include <string>
+#include "TestLayeredLBStrategy.h"
+#include "GameFramework/DefaultPawn.h"
+#include "Misc/Optional.h"
+
+
+#define LAYEREDLBSTRATEGY_TEST(TestName) \
+	GDK_TEST(Core, ULayeredLBStrategy, TestName)
+
+
+
+namespace
+{
```

Comment:
We don't use indents within namespaces (IDEs make this hard to do sometimes I realise 😅 )

Reply:
ah yes true, done